### PR TITLE
Align values with update to brand color

### PR DESCRIPTION
**Pull request by anthonydillon**
# Done
- Aligned the values
- Updated the brand color

## QA
- Check all values line up in global-settings
- Check the header background is #F5F5F5

### CODE REVIEW

File: scss/_global-settings.scss [hunk 1]
```diff
@@ -7,11 +7,11 @@ $asset-path: 'https://assets.ubuntu.com/sites/ubuntu/latest/u/';
 /// transparent to use throughout the site
 $transparent:           transparent !default;
 /// the theme's core brand colour
-$brand-color:           #efefef !default;
+$brand-color:           #f5f5f5 !default;
 /// light brand colour
 $brand-color-light:     lighten($brand-color, 48%) !default; ///TODO make this a lighter brand color
 /// header link color
-$header-link-color:	#333 !default;
+$header-link-color:		#333 !default;
```
Comment (nottrobin) [inline scss/_global-settings.scss]: Introduced tabs here...

File: scss/_global-settings.scss [hunk 2]
```diff
@@ -64,19 +64,19 @@ $gutter-width:      	20px !default;
 $columns:           	12 !default;
 
 /// mobile nav
-$mobile-nav:     	#333 !default;
+$mobile-nav:     		#333 !default;
```
Comment (nottrobin) [inline scss/_global-settings.scss]: and here

File: scss/_global-settings.scss [hunk 3]
```diff
@@ -64,19 +64,19 @@ $gutter-width:      	20px !default;
...
 
 /// nav active background
 $nav-active-bg:     	#ddd !default;
 
 // Site maximum width
-$site-max-width: 	1030px !default;
+$site-max-width: 		1030px !default;
```
Comment (nottrobin) [inline scss/_global-settings.scss]: and here

File: scss/_global-settings.scss [hunk 4]
```diff
@@ -64,19 +64,19 @@ $gutter-width:      	20px !default;
...
 
 /// Medium breakpoint
 $breakpoint-medium: 	768px !default;
 
 /// Large breakpoint
-$breakpoint-large: 	$site-max-width !default;
+$breakpoint-large: 		$site-max-width !default;
```
Comment (nottrobin) [inline scss/_global-settings.scss]: and here

File: scss/_global-settings.scss [hunk 5]
```diff
@@ -7,11 +7,11 @@ $asset-path: 'https://assets.ubuntu.com/sites/ubuntu/latest/u/';
 /// transparent to use throughout the site
 $transparent:           transparent !default;
 /// the theme's core brand colour
-$brand-color:           #efefef !default;
+$brand-color:           #f5f5f5 !default;
 /// light brand colour
 $brand-color-light:     lighten($brand-color, 48%) !default; ///TODO make this a lighter brand color
```
Comment (nottrobin) [inline scss/_global-settings.scss]: This TODO comment here - has this been done by you just now? Or what? What did the comment mean? Maybe you should remove it - or fix it?

**Comment by nottrobin** [comment]
I made some comments. Let me know when they're done I'll have a final little look.

**Comment by anthonydillon** [comment]
@nottrobin fixed up the issues your spotted. Can you re-review please?

**Comment by nottrobin** [comment]
Yep looks good, :+1:

**Comment by anthonydillon** [comment]
@nottrobin thanks

**Pull request merged**